config/config.exs

```diff
@@ -23,6 +23,4 @@ use Mix.Config
 
 config :phoenix, json_library: Jason
 
-if Mix.env() == :test do
-  import_config "test.exs"
-end
+import_config "#{Mix.env()}.exs"
```

Comment:
This will break whenever `Mix.env()` is not `:test`, as we do not have `dev.exs` or `prod.exs` files.

Reply:
Maybe we can just remove `test.exs` completely? It's just setting the logger level to `:info` in tests, so we can put that line here in `config.exs`.